defaults/main.yml

```diff
@@ -16,7 +16,7 @@ apache_global_vhost_settings: |
   DirectoryIndex index.php index.html
 
 apache_vhosts:
-  # Additional properties: 'serveradmin, serveralias, extra_parameters'.
+  # Additional properties: 'serveradmin, serveralias, directory_extra, extra_parameters'.
```

Comment:
I'll make a deal, I'll change directory_extra -->
extra_parameters_directory if you'll make geerlingguy.nginx.vhost.
*server_name* consistent with geerlingguy.apache.vhost.*servername*

That was a cut/paste waiting to happen... :-)

I changed it up.

On Mon, May 9, 2016 at 11:08 PM, Jeff Geerling <notifications@github.com>
wrote:

> In defaults/main.yml
> <https://github.com/geerlingguy/ansible-role-apache/pull/88#discussion_r62606718>
> :
>
> > @@ -16,7 +16,7 @@ apache_global_vhost_settings: |
> >    DirectoryIndex index.php index.html
> >
> >  apache_vhosts:
> > -  # Additional properties: 'serveradmin, serveralias, extra_parameters'.
> > +  # Additional properties: 'serveradmin, serveralias, directory_extra, extra_parameters'.
>
> Can you call it extra_parameters_directory instead?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/geerlingguy/ansible-role-apache/pull/88/files/5a61939b51e75bbd8597bee474665a1e0ceeb074#r62606718>
>



-- 
Kind Regards,
Greg Kedge
202.603.0648
gregwork@kedges.com